WalletWasabi/Blockchain/TransactionBuilding/BnB/CheapestSelectionStrategy.cs

```diff
@@ -10,16 +10,18 @@ namespace WalletWasabi.Blockchain.TransactionBuilding.BnB;
 public class CheapestSelectionStrategy
 {
 	private long _currentInputCosts = 0;
-	private long _bestTargetSoFar = long.MaxValue;
+	private long _bestTargetSoFar;
 	private long[]? _bestSelectionSoFar;
 
 	/// <param name="target">Value in satoshis.</param>
 	/// <param name="inputCosts">Costs of spending coins in satoshis.</param>
-	public CheapestSelectionStrategy(long target, long[] inputValues, long[] inputCosts)
+	public CheapestSelectionStrategy(long target, SuggestionType suggestionType, long[] inputValues, long[] inputCosts)
```

Comment:
You can switch parameter order as  `SuggestionType` is the most important here.

```suggestion
	public CheapestSelectionStrategy(SuggestionType suggestionType, long target, long[] inputValues, long[] inputCosts)
```

Reply:
(You can do it with a few clicks using Visual studio. CTRL+. and change method signature)